task/powershell/0.1/powershell.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: powershell
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: powershell, pwsh
+    tekton.dev/displayName: powershell
+spec:
+  description: >-
+    This task will run powershell commands
+  params:
+    - name: tag
+      description: The tag for the Powershell image
+      type: string
+      default: "latest"
+    - name: command
+      description: The Powershell command
+      type: string
+      default: "Write-Output 'Please use command parameter to enter'"
+    - name: verbose
+      description: Verbosity level for command
+      type: string
+      default: "SilentlyContinue"
+  steps:
+    - name: invoke-script
+      image: mcr.microsoft.com/powershell:$(inputs.params.tag)
+      script: |
+        #!/usr/bin/env pwsh
+        
+        $VerbosePreference = "$(inputs.params.verbose)"
+        $command = "$(inputs.params.command)"
```

Comment:
can we run multiple commands or is it that we can run only one command at a time?

Reply:
Yes you can run many commands, I have just updated README.md

[README.md](https://github.com/haidouks/catalog/blob/master/task/powershell/0.1/README.md)